cloud/monitoring.rst

```diff
@@ -40,6 +40,11 @@ PostgreSQL Write-Ahead Log
 * WAL Bytes Written / Second
    .. image:: ../images/metric-wal.png
 
+Cloud Platform Status
+=====================
+
+`Status.citusdata.com <https://status.citusdata.com/>`_ provides the current status and incident history report for the Citus Cloud platform.
```

Comment:
Is there a reason this is part of this PR? Given these PR's typically take some time, I'd be in favor of just opening up a small PR and getting it merged right away.

Reply:
I threw that in after an in-person conversation, since I was already working in this file. But I can move it into its own dedicated PR to make this PR a little cleaner.